src/probnum/diffeq/odefiltsmooth/ivpfiltsmooth.py

```diff
@@ -164,4 +168,75 @@ def _estimate_diffusion(self, pred_rv, meas_rv):
 
     @property
     def prior(self):
-        return self.gfilt.dynamicmodel
+        return self.gfilt.dynamic_model
+
+
+#####################################################################################################3
+#
+# below the code for using preconditioning in step():
+#
+# It seems slower though and we don't need it for stability in the forward pass?
+#
+#####################################################################################################3
```

Comment:
This. The ODESolver.step() is not using the preconditioner currently. I implemented it and it turned out that it was a bit slower than not using it (110-120% of the runtime). Since we only NEED it in the smoothing iteration, I'd suggest sticking to this for now? What do you think?

Reply:
I would guess that I screwed something up in the uncertainty calibration. It is probably just as fast, though probably not faster, and since it is not needed here for stability reasons I would argue leaving it out for code readability and clarity

Shall I remove the commented code then? We can always go back to trying it out here in the future if we like?

Reply:
Sounds good!